mne_bids/utils.py

```diff
@@ -353,16 +354,16 @@ def _write_json(dictionary, fname, overwrite=False, verbose=False):
         print(json_output)
 
 
-def _write_tsv(fname, df, overwrite=False, verbose=False):
-    """Write dataframe to a .tsv file."""
+def _write_tsv(dictionary, fname, overwrite=False, verbose=False):
+    """Write an ordered dictionary to a .tsv file"""
     if op.exists(fname) and not overwrite:
         raise FileExistsError('"%s" already exists. Please set '
                               'overwrite to True.' % fname)
-    df.to_csv(fname, sep='\t', index=False, na_rep='n/a', encoding='utf-8')
```

Comment:
hmm ... what happens to `'n/a'` now?

Reply:
I think it's fine because we can't construct the OrderedDictionary without having some value there.
The values will default to `n/a` in when populating the lists that are written to the tsv file.